src/server/email/sendEmail.js

```diff
@@ -60,7 +62,7 @@ export default async function sendEmailPromise(to, template, props) {
     to,
     message: message.toString('ascii')
   };
-  if (!getMailgunApiConfig().apiKey) {
+  if (!supressConsole && !getMailgunApiConfig().apiKey) {
```

Comment:
This makes testing much more bearable. Alternatively, I could have used the environment variable. However, I felt adding this option might be convenient in the future.